tests/Aspire.Cli.Tests/Hosting/CliOrphanDetectorTests.cs

```diff
@@ -92,34 +88,34 @@ public async Task CliOrphanDetectorAfterTheProcessWasRunningForAWhileThenStops()
         Assert.True(await stopSignalChannel.Reader.WaitToReadAsync());
     }
 
-    [Fact]
-    [QuarantinedTest("https://github.com/dotnet/aspire/issues/7920")]
-    public async Task AppHostExitsWhenCliProcessPidDies()
-    {
-        using var fakeCliProcess = RemoteExecutor.Invoke(
-            static () => Thread.Sleep(Timeout.Infinite),
-            new RemoteInvokeOptions { CheckExitCode = false }
-            );
+    // [Fact]
```

Comment:
??

Reply:
Adding back in, removed it temporarily whilst debugging an agent mode issue.